vagrant-kubernetes-istio/RunTestOnHost/Vagrantfile

```diff
@@ -75,10 +75,13 @@ Vagrant.configure("2") do |config|
   config.vm.provision "shell", inline: <<-SHELL
      echo "provisioning vagrant vm..."
      apt-get update
+     echo "install git"
```

Comment:
why do you need git?

Reply:
go get depends on git, I get error complaining git is missed. I find the solution here.
https://stackoverflow.com/questions/36044275/golang-go-get-command-showing-go-missing-git-command-error

Reply:
ok.. I didn't see that but that could also be the reason why I was getting some packages missing error..